clients/ops/admin-ui/src/app/store.ts

```diff
@@ -38,14 +73,33 @@ const reducer = {
   connectionType: connectionTypeReducer,
 };
 
+const rootReducer = (state: any, action: AnyAction) => {
+  if (action.type === "auth/logout") {
+    storage.removeItem(STORAGE_ROOT_KEY);
+    // eslint-disable-next-line no-param-reassign
+    state = {};
+  }
+  return combineReducers(reducer)(state, action);
+};
+
+const persistConfig = {
```

Comment:
You definitely want to [blacklist](https://redux-toolkit.js.org/usage/usage-guide#use-with-redux-persist) all API reducers. 
![Screen Shot 2022-09-30 at 12 07 55](https://user-images.githubusercontent.com/2236777/193340133-5a4c5eb4-ea95-4344-bf91-aa1e2ea9d50b.png)

RTK Query uses the state to keep track of subscriptions and networking, which are all invalidated when the page reloads. Persisting them would mean a bunch of cleanup happens unnecessarily on load.

(Probably worth a code comment describing how adding a new API reducer should also add it to the blacklist.)